app/retail/models.py

```diff
@@ -0,0 +1,81 @@
+from django.db import models
+
+from economy.models import SuperModel
+from github.utils import get_user
+from rest_framework import serializers
+
+
+class Idea(SuperModel):
+    """Define the structure of idea"""
+    full_name = models.CharField(max_length=255)
+    email = models.EmailField(max_length=255)
+    github_username = models.CharField(max_length=255)
```

Comment:
I think it might be better to reference profile here in order to remove redundancy but I see avatar_url spread across models. What do You think? Maybe refactor in another PR ?

Reply:
@kziemianek Ideally, if we could reference `profile` and add a related name for reverse as: `ideas`, that would be 👍